spec.bs

```diff
@@ -3770,6 +3794,19 @@ To <dfn>convert GenerateBidOutput to generated bid</dfn> given a {{GenerateBidOu
       1. If [=finding matching ad=] given |componentUrl|, |ig|, and true returns null, return failure.
       1. [=list/Append=] |componentDescriptor| to |adComponentDescriptors|.
     1. Set |bid|'s [=generated bid/ad component descriptors=] to |adComponentDescriptors|.
+  1. If |generateBidOutput|["{{GenerateBidOutput/targetNumAdComponents}}"] [=map/exists=]:
+    1. If |generateBidOutput|["{{GenerateBidOutput/targetNumAdComponents}}"] = 0 or
+       |generateBidOutput|["{{GenerateBidOutput/targetNumAdComponents}}"] &gt; 40,
+       return failure.
+    1. If |bid|'s [=generated bid/ad component descriptors=] is null, or
+          |bid|'s [=generated bid/ad component descriptors=]'s [=list/size=] &lt;
+          |generateBidOutput|["{{GenerateBidOutput/targetNumAdComponents}}"], return failure.
+    1. If |generateBidOutput|["{{GenerateBidOutput/targetNumAdComponents}}"] &lt;

```

Comment:
`&le;` ?
https://crsrc.org/c/content/services/auction_worklet/set_bid_bindings.cc;drc=a169ebabcd533a7f5489ac17c1df80f8b89c6fee;l=605

Reply:
Nope? The order of arguments is other way around?